posts/rails_memory.md

```diff
@@ -3,6 +3,8 @@ title: "Rails memory leak problems"
 date: "2021-10-10"
 ---
 
+Checking....
+
 Ruby memory leak problems are maybe familiar with Rubyist and you can find so many articles about them palso the solution with detailed explanation. As a Rails developer, I also stuck on that, and I want to share my experience. Let's start!
```

Comment:
- familiar with -> familiar to
  - <somebody>'s familiar **with** something
  - <something>'s familiar **to** somebody
- palso??
- detailed explanation -> detailed explanations or **a** detailed explanation
- I also stuck -> I'm stuck. Better way to say: I myself have also encountered this problem before

Reply:
https://github.com/HoangNguyen679/aphelios/pull/1/commits/5304e5be3281d5c2a0e438f5efdab526f961704e fixed